### PR TITLE
Bitnami based container

### PR DESCRIPTION
**Pull request by jbianquetti-nami**
The Dockerfile has been migrated to Bitnami one.

I substituted the default Dockerfile for a one which will allow us to test it in CicleCI.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+
+COPY . /app
+
+WORKDIR /app
+
+RUN yarn install
+
+RUN npm rebuild node-sass
+
+RUN yarn run compile
```
Comment (sebgoa) [inline Dockerfile]: can you combine RUN directives in a single one with \ (that's best practice)

Comment (jbianquetti-nami) [inline Dockerfile]: Sure. I've split into different steps to be easier to debug, but now it makes sense rejoin all them.

**Comment by chipironcin** [comment]
Good job! It was already using a Bitnami base image but this is a good revamp!

**Comment by sebgoa** [comment]
ok did not see the circle ci script

**Pull request merged**